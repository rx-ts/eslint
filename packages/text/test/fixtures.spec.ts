import path from 'node:path'

import { ESLint } from 'eslint'

const eslint = new ESLint({
  useEslintrc: false,
  ignore: false,
  overrideConfig: {
    overrides: [
      {
        files: '*.txt',
        extends: 'plugin:text/recommended',
      },
    ],
  },
})

test('fixtures', async () => {
  const results = await eslint.lintFiles(
    path.resolve(__dirname, './fixtures/*.txt'),
  )
  for (const { filePath, messages } of results) {
    expect(messages).toMatchSnapshot(path.basename(filePath))
  }
})
