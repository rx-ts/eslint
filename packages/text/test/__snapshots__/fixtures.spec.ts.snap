// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fixtures: basic.txt 1`] = `[]`;
